Hackathon_2024.10/60 Autorisation/Search VVT by name.bru

```diff
@@ -20,6 +20,6 @@ vars:post-response {
 }
 
 assert {
-  res.body.length: eq 1
```

Comment:
In the local setup this should only return 1 item right? What are your results?

Reply:
A search without query params returned duplicate results. https://github.com/nuts-foundation/nuts-node/issues/3455, did you perhaps change the query too?

Reply:
res.body.length is the length of the entire body in bytes, right?

Reply:
res.body.length is the number of results for the search. Depending on your search query and what other parties registered on the discovery service it may have returned more than 1 result

Reply:
Ok, not sure what happened then. nvm ;-)